www/src/html.js

```diff
@@ -72,6 +72,8 @@ export default class HTML extends React.Component {
             href={`/safari-pinned-tab.svg`}
             color="#5bbad5"
           />
+          <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
```

Comment:
Could you move this to after the body? No need to block loading for this.

Reply:
Is there a way to load glamor's global styles after the body too? Otherwise it gets quite fiddly overriding the existing DocSearch styles.